crates/core/src/lib.rs

```diff
@@ -37,7 +37,7 @@ pub use io::OutputBuffer;
 pub use store::{Store, StoreBuilder, Wasi, WasiVersion};
 
 /// The default [`EngineBuilder::epoch_tick_interval`].
-pub const DEFAULT_EPOCH_TICK_INTERVAL: Duration = Duration::from_millis(10);
+pub const DEFAULT_EPOCH_TICK_INTERVAL: Duration = Duration::from_micros(100);
```

Comment:
This arbitrarily updates the original arbitrary value, which "felt too slow" to me. :shrug: